OracleTuxedo/core/dockerfiles/12.2.2/Dockerfile

```diff
@@ -9,13 +9,13 @@
 #   sh build.sh
 
 # Pull base image
-FROM oracle/serverjre:8
+FROM container-registry.oracle.com/java/serverjre
```

Comment:
This should be changed to `oracle/serverjre:8` as all Dockerfiles in this git repo are based on other manually built images from the same repo.